app/isomorphic/components/layouts/app-logo/index.js

```diff
@@ -4,19 +4,17 @@ import { Link } from "@quintype/components";
 import "./app-logo.m.css";
 
 const AppLogo = () => (
-  <h1 className="logo-wrapper">
-    <Link href="/">
-      <img
-        height="36"
-        width="178"
-        loading="lazy"
-        styleName="publisher-logo"
-        srcSet="https://qtstage-01.gumlet.io/malibu/2021-03/193b1a09-89cf-4b07-8d4f-fcf00d534577/malibu_logo_new.svg"
-        data-src="https://qtstage-01.gumlet.io/malibu/2021-03/193b1a09-89cf-4b07-8d4f-fcf00d534577/malibu_logo_new.svg"
-        alt="Logo"
-      />
-    </Link>
-  </h1>
+  <Link href="/">
+    <img
+      height="36"
+      width="178"
+      loading="lazy"
```

Comment:
if your lazy loading this image do you have a fallback image for this ?

Reply:
why do we need a fallback for the app logo? 
The loading attribute on an <img> element (or the loading attribute on an <iframe>) can be used to instruct the browser to defer loading of images/iframes that are off-screen until the user scrolls near them.